extensions/wikia/AjaxPoll/AjaxPoll_body.php

```diff
@@ -20,14 +20,6 @@ class AjaxPollClass {
 	public static $mCount = 0; # macbre: count ajax polls on the page
```

Comment:
Changes in this class are related to [SOC-410](https://wikia-inc.atlassian.net/browse/SOC-410)